src/features/streamlabs/errors/streamlabs/http-status-error.ts

```diff
@@ -0,0 +1,5 @@
+import { StreamlabsError } from "./streamlabs-error";
+
+export class HttpStatusError extends StreamlabsError {
+  override name = "HttpStatusError";
```

Comment:
First time I saw that override keyword in TypeScript, I thought it was only available in c#

Reply:
The only reason I'm even adding the name, is because `instanceof` does not work when there are multiple instances of the same module in memory. 

`Node` uses the [`ErrnoException`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/globals.d.ts#L194) which extends the standard `Error` with a `code` property to do exaactly the same thing.